<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1333</width>
    <height>842</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Gestion des Clients</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QPushButton" name="ajouter_Materiel">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>540</y>
      <width>171</width>
      <height>61</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 600 italic 11pt &quot;Segoe UI&quot;;
border-color: rgb(0, 0, 139); /* Bleu roi */
border: 2px solid rgb(0, 0, 139); /* Bordure bleu roi */
padding: 10px 20px; /* Optionnel, pour ajouter de l'espace à l'intérieur du bouton */
background-color: white; /* Optionnel, définit la couleur de fond du bouton */
color: rgb(255, 85, 127); /* Rose pour le texte */
QPushButton {
    border-radius: 50px;
    background-color: #F0F0F0; /* Couleur de fond */
    border: 2px solid #000; /* Bordure */
}</string>
    </property>
    <property name="text">
     <string>Ajouter</string>
    </property>
   </widget>
   <widget class="QTableView" name="affichage">
    <property name="geometry">
     <rect>
      <x>430</x>
      <y>30</y>
      <width>611</width>
      <height>441</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
   </widget>
   <widget class="QLineEdit" name="id_supp">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>610</y>
      <width>141</width>
      <height>21</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_9">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>630</y>
      <width>63</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Idmatériel</string>
    </property>
   </widget>
   <widget class="QPushButton" name="supression_Materiel">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>540</y>
      <width>171</width>
      <height>61</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 600 italic 11pt &quot;Segoe UI&quot;;
border-color: rgb(0, 0, 139); /* Bleu roi */
border: 2px solid rgb(0, 0, 139); /* Bordure bleu roi */
padding: 10px 20px; /* Optionnel, pour ajouter de l'espace à l'intérieur du bouton */
background-color: white; /* Optionnel, définit la couleur de fond du bouton */
color: rgb(255, 85, 127); /* Rose pour le texte */</string>
    </property>
    <property name="text">
     <string>supression</string>
    </property>
   </widget>
   <widget class="QPushButton" name="modifier_Materiel">
    <property name="geometry">
     <rect>
      <x>370</x>
      <y>540</y>
      <width>171</width>
      <height>61</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 600 italic 11pt &quot;Segoe UI&quot;;
 border-color: rgb(0, 0, 139); /* Bleu roi */
border: 2px solid rgb(0, 0, 139); /* Bordure bleu roi */
padding: 10px 20px; /* Optionnel, pour ajouter de l'espace à l'intérieur du bouton */
background-color: white; /* Optionnel, définit la couleur de fond du bouton */
color: rgb(255, 85, 127); /* Rose pour le texte */</string>
    </property>
    <property name="text">
     <string>modifier</string>
    </property>
   </widget>
   <widget class="QPushButton" name="recherche">
    <property name="geometry">
     <rect>
      <x>550</x>
      <y>540</y>
      <width>141</width>
      <height>61</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 600 italic 11pt &quot;Segoe UI&quot;;
border-color: rgb(0, 0, 139); /* Bleu roi */
border: 2px solid rgb(0, 0, 139); /* Bordure bleu roi */
padding: 10px 20px; /* Optionnel, pour ajouter de l'espace à l'intérieur du bouton */
background-color: white; /* Optionnel, définit la couleur de fond du bouton */
color: rgb(255, 85, 127); /* Rose pour le texte */</string>
    </property>
    <property name="text">
     <string>Rechercher</string>
    </property>
   </widget>
   <widget class="QComboBox" name="comboBox_rech">
    <property name="geometry">
     <rect>
      <x>550</x>
      <y>610</y>
      <width>131</width>
      <height>21</height>
     </rect>
    </property>
    <item>
     <property name="text">
      <string>idmateriel</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Nom</string>
     </property>
    </item>
   </widget>
   <widget class="QLineEdit" name="id_rech">
    <property name="geometry">
     <rect>
      <x>390</x>
      <y>610</y>
      <width>131</width>
      <height>21</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="trier_Materiel">
    <property name="geometry">
     <rect>
      <x>1030</x>
      <y>540</y>
      <width>141</width>
      <height>61</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">
font: 600 italic 11pt &quot;Segoe UI&quot;;
border-color: rgb(0, 0, 139); /* Bleu roi */
border: 2px solid rgb(0, 0, 139); /* Bordure bleu roi */
padding: 10px 20px; /* Optionnel, pour ajouter de l'espace à l'intérieur du bouton */
background-color: white; /* Optionnel, définit la couleur de fond du bouton */
color: rgb(255, 85, 127); /* Rose pour le texte */</string>
    </property>
    <property name="text">
     <string>trier</string>
    </property>
   </widget>
   <widget class="QComboBox" name="comboBox_trie">
    <property name="geometry">
     <rect>
      <x>1030</x>
      <y>610</y>
      <width>141</width>
      <height>20</height>
     </rect>
    </property>
    <item>
     <property name="text">
      <string>Date d'achat</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Etat</string>
     </property>
    </item>
   </widget>
   <widget class="QPushButton" name="btnExporterPDF">
    <property name="geometry">
     <rect>
      <x>710</x>
      <y>540</y>
      <width>151</width>
      <height>61</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 600 italic 11pt &quot;Segoe UI&quot;;
border-color: rgb(0, 0, 139); /* Bleu roi */
border: 2px solid rgb(0, 0, 139); /* Bordure bleu roi */
padding: 10px 20px; /* Optionnel, pour ajouter de l'espace à l'intérieur du bouton */
background-color: white; /* Optionnel, définit la couleur de fond du bouton */
color: rgb(255, 85, 127); /* Rose pour le texte */</string>
    </property>
    <property name="text">
     <string>Pdf</string>
    </property>
   </widget>
   <widget class="QPushButton" name="statsButton">
    <property name="geometry">
     <rect>
      <x>880</x>
      <y>540</y>
      <width>141</width>
      <height>61</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 600 italic 11pt &quot;Segoe UI&quot;;

  border-color: rgb(0, 0, 139); /* Bleu roi */
border: 2px solid rgb(0, 0, 139); /* Bordure bleu roi */
padding: 10px 20px; /* Optionnel, pour ajouter de l'espace à l'intérieur du bouton */
background-color: white; /* Optionnel, définit la couleur de fond du bouton */
color: rgb(255, 85, 127); /* Rose pour le texte */</string>
    </property>
    <property name="text">
     <string>statistique</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="dest">
    <property name="geometry">
     <rect>
      <x>1190</x>
      <y>610</y>
      <width>131</width>
      <height>20</height>
     </rect>
    </property>
    <property name="placeholderText">
     <string>Adresse destinataire </string>
    </property>
   </widget>
   <widget class="QPushButton" name="mail">
    <property name="geometry">
     <rect>
      <x>1190</x>
      <y>540</y>
      <width>131</width>
      <height>61</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 600 italic 11pt &quot;Segoe UI&quot;;
border-color: rgb(0, 0, 139); /* Bleu roi */
border: 2px solid rgb(0, 0, 139); /* Bordure bleu roi */
padding: 10px 20px; /* Optionnel, pour ajouter de l'espace à l'intérieur du bouton */
background-color: white; /* Optionnel, définit la couleur de fond du bouton */
color: rgb(255, 85, 127); /* Rose pour le texte */</string>
    </property>
    <property name="text">
     <string>Notifier </string>
    </property>
   </widget>
   <widget class="QLabel" name="qrcodeLabel">
    <property name="geometry">
     <rect>
      <x>620</x>
      <y>640</y>
      <width>151</width>
      <height>121</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="qrcodegen">
    <property name="geometry">
     <rect>
      <x>780</x>
      <y>700</y>
      <width>141</width>
      <height>51</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 600 italic 11pt &quot;Segoe UI&quot;;
border-color: rgb(0, 0, 139); /* Bleu roi */
border: 2px solid rgb(0, 0, 139); /* Bordure bleu roi */
padding: 10px 20px; /* Optionnel, pour ajouter de l'espace à l'intérieur du bouton */
background-color: white; /* Optionnel, définit la couleur de fond du bouton */
color: rgb(255, 85, 127); /* Rose pour le texte */</string>
    </property>
    <property name="text">
     <string>qrcode</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_12">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>-10</y>
      <width>1591</width>
      <height>911</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap>../../../Pictures/Screenshots/Capture d'écran 2024-11-21 233216.png</pixmap>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="label_11">
    <property name="geometry">
     <rect>
      <x>1160</x>
      <y>30</y>
      <width>221</width>
      <height>441</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap>../../../Pictures/Screenshots/Capture d'écran 2024-11-22 000039.png</pixmap>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QStackedWidget" name="stackedWidget">
    <property name="geometry">
     <rect>
      <x>-10</x>
      <y>10</y>
      <width>441</width>
      <height>481</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: transparent;</string>
    </property>
    <property name="currentIndex">
     <number>0</number>
    </property>
    <widget class="QWidget" name="page">
     <widget class="QLineEdit" name="duree_de_vie_materiel">
      <property name="geometry">
       <rect>
        <x>280</x>
        <y>410</y>
        <width>113</width>
        <height>26</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">QTableView {
    color: rgb(175, 238, 238); /* Bleu clair */
    background-color: rgb(170, 170, 255); /* Reste inchangé */
}

QLineEdit {
    color: rgb(255, 255, 255); /* Blanc */
    background-color: rgb(175, 238, 238); /* Bleu clair */
}
</string>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QLineEdit" name="nom_materiel">
      <property name="geometry">
       <rect>
        <x>280</x>
        <y>110</y>
        <width>113</width>
        <height>26</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">QTableView {
    color: rgb(175, 238, 238); /* Bleu clair */
    background-color: rgb(170, 170, 255); /* Reste inchangé */
}

QLineEdit {
    color: rgb(255, 255, 255); /* Blanc */
    background-color: rgb(175, 238, 238); /* Bleu clair */
}

</string>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QLineEdit" name="prix_materiel">
      <property name="geometry">
       <rect>
        <x>280</x>
        <y>310</y>
        <width>113</width>
        <height>26</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">QTableView {
    color: rgb(175, 238, 238); /* Bleu clair */
    background-color: rgb(170, 170, 255); /* Reste inchangé */
}

QLineEdit {
    color: rgb(255, 255, 255); /* Blanc */
    background-color: rgb(175, 238, 238); /* Bleu clair */
}</string>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QLineEdit" name="etat_materiel">
      <property name="geometry">
       <rect>
        <x>280</x>
        <y>160</y>
        <width>113</width>
        <height>26</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">QTableView {
    color: rgb(175, 238, 238); /* Bleu clair */
    background-color: rgb(170, 170, 255); /* Reste inchangé */
}

QLineEdit {
    color: rgb(255, 255, 255); /* Blanc */
    background-color: rgb(175, 238, 238); /* Bleu clair */
}</string>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QLineEdit" name="duree_garantie_materiel">
      <property name="geometry">
       <rect>
        <x>280</x>
        <y>260</y>
        <width>113</width>
        <height>26</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">QTableView {
    color: rgb(175, 238, 238); /* Bleu clair */
    background-color: rgb(170, 170, 255); /* Reste inchangé */
}

QLineEdit {
    color: rgb(255, 255, 255); /* Blanc */
    background-color: rgb(175, 238, 238); /* Bleu clair */
}
</string>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QLabel" name="label_5">
      <property name="geometry">
       <rect>
        <x>50</x>
        <y>260</y>
        <width>63</width>
        <height>20</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">font: 900 italic 12pt &quot;Segoe UI&quot;;
color: rgb(255, 85, 127);</string>
      </property>
      <property name="text">
       <string>Duree Garantie</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_7">
      <property name="geometry">
       <rect>
        <x>50</x>
        <y>360</y>
        <width>211</width>
        <height>31</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">font: 900 italic 12pt &quot;Segoe UI&quot;;
color: rgb(255, 85, 127);</string>
      </property>
      <property name="text">
       <string>Quantite disponible</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_10">
      <property name="geometry">
       <rect>
        <x>40</x>
        <y>30</y>
        <width>371</width>
        <height>441</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(32, 78, 127);</string>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QLabel" name="label_2">
      <property name="geometry">
       <rect>
        <x>50</x>
        <y>100</y>
        <width>81</width>
        <height>41</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">font: 900 italic 12pt &quot;Segoe UI&quot;;
color: rgb(255, 85, 127);</string>
      </property>
      <property name="text">
       <string>Nom</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_6">
      <property name="geometry">
       <rect>
        <x>50</x>
        <y>310</y>
        <width>63</width>
        <height>20</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">font: 900 italic 12pt &quot;Segoe UI&quot;;
color: rgb(255, 85, 127);</string>
      </property>
      <property name="text">
       <string>Prix</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="date_achat_materiel">
      <property name="geometry">
       <rect>
        <x>280</x>
        <y>210</y>
        <width>113</width>
        <height>26</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">QTableView {
    color: rgb(175, 238, 238); /* Bleu clair */
    background-color: rgb(170, 170, 255); /* Reste inchangé */
}

QLineEdit {
    color: rgb(255, 255, 255); /* Blanc */
    background-color: rgb(175, 238, 238); /* Bleu clair */
}</string>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QLabel" name="label_3">
      <property name="geometry">
       <rect>
        <x>50</x>
        <y>160</y>
        <width>63</width>
        <height>20</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">font: 900 italic 12pt &quot;Segoe UI&quot;;
color: rgb(255, 85, 127);</string>
      </property>
      <property name="text">
       <string>Etat</string>
      </property>
     </widget>
     <widget class="QLabel" name="label">
      <property name="geometry">
       <rect>
        <x>50</x>
        <y>60</y>
        <width>121</width>
        <height>20</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">font: 900 italic 12pt &quot;Segoe UI&quot;;
color: rgb(255, 85, 127);</string>
      </property>
      <property name="text">
       <string>Idmatériel</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_8">
      <property name="geometry">
       <rect>
        <x>50</x>
        <y>420</y>
        <width>201</width>
        <height>16</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">font: 900 italic 12pt &quot;Segoe UI&quot;;
color: rgb(255, 85, 127);
</string>
      </property>
      <property name="text">
       <string>Duree de vie</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_4">
      <property name="geometry">
       <rect>
        <x>50</x>
        <y>210</y>
        <width>151</width>
        <height>21</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">font: 900 italic 12pt &quot;Segoe UI&quot;;
color: rgb(255, 85, 127);</string>
      </property>
      <property name="text">
       <string>Date d'achat</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="id_materiel">
      <property name="geometry">
       <rect>
        <x>280</x>
        <y>60</y>
        <width>113</width>
        <height>26</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">QTableView {
    color: rgb(175, 238, 238); /* Bleu clair */
    background-color: rgb(170, 170, 255); /* Reste inchangé */
}

QLineEdit {
    color: rgb(255, 255, 255); /* Blanc */
    background-color: rgb(175, 238, 238); /* Bleu clair */
}
</string>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QLineEdit" name="quantite_materiel">
      <property name="geometry">
       <rect>
        <x>280</x>
        <y>360</y>
        <width>113</width>
        <height>26</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">QTableView {
    color: rgb(175, 238, 238); /* Bleu clair */
    background-color: rgb(170, 170, 255); /* Reste inchangé */
}

QLineEdit {
    color: rgb(255, 255, 255); /* Blanc */
    background-color: rgb(175, 238, 238); /* Bleu clair */
}
</string>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <zorder>label_10</zorder>
     <zorder>duree_de_vie_materiel</zorder>
     <zorder>nom_materiel</zorder>
     <zorder>prix_materiel</zorder>
     <zorder>etat_materiel</zorder>
     <zorder>duree_garantie_materiel</zorder>
     <zorder>label_5</zorder>
     <zorder>label_7</zorder>
     <zorder>label_2</zorder>
     <zorder>label_6</zorder>
     <zorder>date_achat_materiel</zorder>
     <zorder>label_3</zorder>
     <zorder>label</zorder>
     <zorder>label_8</zorder>
     <zorder>label_4</zorder>
     <zorder>id_materiel</zorder>
     <zorder>quantite_materiel</zorder>
    </widget>
    <widget class="QWidget" name="page_2">
     <widget class="QLabel" name="label_13">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>50</y>
        <width>121</width>
        <height>20</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">font: 900 italic 12pt &quot;Segoe UI&quot;;
color: rgb(255, 85, 127);</string>
      </property>
      <property name="text">
       <string>Idmatériel</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="id_materiel_2">
      <property name="geometry">
       <rect>
        <x>260</x>
        <y>50</y>
        <width>113</width>
        <height>26</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">QTableView {
    color: rgb(175, 238, 238); /* Bleu clair */
    background-color: rgb(170, 170, 255); /* Reste inchangé */
}

QLineEdit {
    color: rgb(255, 255, 255); /* Blanc */
    background-color: rgb(175, 238, 238); /* Bleu clair */
}
</string>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QLabel" name="label_14">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>20</y>
        <width>371</width>
        <height>441</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(32, 78, 127);</string>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QPushButton" name="ajouter_Materiel_2">
      <property name="geometry">
       <rect>
        <x>40</x>
        <y>370</y>
        <width>131</width>
        <height>51</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">font: 600 italic 11pt &quot;Segoe UI&quot;;
border-color: rgb(0, 0, 139); /* Bleu roi */
border: 2px solid rgb(0, 0, 139); /* Bordure bleu roi */
padding: 10px 20px; /* Optionnel, pour ajouter de l'espace à l'intérieur du bouton */
background-color: white; /* Optionnel, définit la couleur de fond du bouton */
color: rgb(255, 85, 127); /* Rose pour le texte */
QPushButton {
    border-radius: 50px;
    background-color: #F0F0F0; /* Couleur de fond */
    border: 2px solid #000; /* Bordure */
}</string>
      </property>
      <property name="text">
       <string>valider</string>
      </property>
     </widget>
     <widget class="QPushButton" name="ajouter_Materiel_3">
      <property name="geometry">
       <rect>
        <x>190</x>
        <y>370</y>
        <width>151</width>
        <height>51</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">font: 600 italic 11pt &quot;Segoe UI&quot;;
border-color: rgb(0, 0, 139); /* Bleu roi */
border: 2px solid rgb(0, 0, 139); /* Bordure bleu roi */
padding: 10px 20px; /* Optionnel, pour ajouter de l'espace à l'intérieur du bouton */
background-color: white; /* Optionnel, définit la couleur de fond du bouton */
color: rgb(255, 85, 127); /* Rose pour le texte */
QPushButton {
    border-radius: 50px;
    background-color: #F0F0F0; /* Couleur de fond */
    border: 2px solid #000; /* Bordure */
}</string>
      </property>
      <property name="text">
       <string>annuler</string>
      </property>
     </widget>
     <zorder>label_14</zorder>
     <zorder>id_materiel_2</zorder>
     <zorder>label_13</zorder>
     <zorder>ajouter_Materiel_2</zorder>
     <zorder>ajouter_Materiel_3</zorder>
    </widget>
   </widget>
   <widget class="QPushButton" name="qrcodegen_2">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>670</y>
      <width>141</width>
      <height>51</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 600 italic 11pt &quot;Segoe UI&quot;;
border-color: rgb(0, 0, 139); /* Bleu roi */
border: 2px solid rgb(0, 0, 139); /* Bordure bleu roi */
padding: 10px 20px; /* Optionnel, pour ajouter de l'espace à l'intérieur du bouton */
background-color: white; /* Optionnel, définit la couleur de fond du bouton */
color: rgb(255, 85, 127); /* Rose pour le texte */</string>
    </property>
    <property name="text">
     <string>verifier</string>
    </property>
   </widget>
   <zorder>label_12</zorder>
   <zorder>ajouter_Materiel</zorder>
   <zorder>id_supp</zorder>
   <zorder>label_9</zorder>
   <zorder>supression_Materiel</zorder>
   <zorder>modifier_Materiel</zorder>
   <zorder>recherche</zorder>
   <zorder>comboBox_rech</zorder>
   <zorder>id_rech</zorder>
   <zorder>trier_Materiel</zorder>
   <zorder>comboBox_trie</zorder>
   <zorder>btnExporterPDF</zorder>
   <zorder>statsButton</zorder>
   <zorder>dest</zorder>
   <zorder>mail</zorder>
   <zorder>qrcodeLabel</zorder>
   <zorder>qrcodegen</zorder>
   <zorder>affichage</zorder>
   <zorder>label_11</zorder>
   <zorder>stackedWidget</zorder>
   <zorder>qrcodegen_2</zorder>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1333</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
